scripts/build-glean-metadata.py

```diff
@@ -269,21 +277,52 @@ def get_app_variant_description(app):
                             )
                         )
                     elif metric_type in SUPPORTED_LOOKER_METRIC_TYPES:
-                        looker_dimension_name = "{}.{}".format(
+                        base_looker_dimension_name = "{}.{}".format(
                             ping_name_snakecase, bigquery_column_name.replace(".", "__")
                         )
-                        looker_metric_link = (
-                            base_looker_explore_link
-                            + "&fields="
-                            + ",".join(
-                                [
-                                    f"{ping_name_snakecase}.submission_date",
-                                    looker_dimension_name,
-                                    f"{ping_name_snakecase}.clients",
-                                ]
+
+                        # For distribution types, we'll aggregate the sum of all distributions per
+                        # day. In most cases, this isn't super meaningful, but provides a starting
```

Comment:
dang, I feel called out 😆

Reply:
I wonder if @acmiyaguchi or @emtwo might have ideas on how to improve this.